spec/javascripts/live_search_spec.js

```diff
@@ -190,6 +192,15 @@ describe("liveSearch", function(){
       expect(liveSearch.updateResults).toHaveBeenCalled();
     });
 
+    it("should update email and atom subscription links when checkbox is changed", function(){
```

Comment:
I've added unit testing for the behaviours of the function itself, at "should replace links with new links when state changes"
I'm half tempted to rename this test, since it doesn't actually test that the links are updated, just that the function is wired up.
As for checking when other types of facet are changed, up to you. I feel like the tests would likely overlap existing testing, but I'm not familiar enough with the existing level of testing to judge that either way with certainty.